### ISSUE TITLE
[BUG]: "Create model from equations" operator throws error and does not return cleaned equations or model

### ISSUE THREAD

**Issue by liunelson** [DARPA-ASKEM/terarium#5691]
- Scenario: Project Collaboration
- Step: 3.2.c
- Link to the project: https://app.staging.terarium.ai/projects/7af5ffef-009d-406a-9270-98429df73dcb/workflow/fd1ebe6c-2e85-4522-9e23-bcef338e79ff?operator=6d01f593-99a9-46b2-88cd-8e88ca8290dc

**Describe the issue**
"Create model from equations" throws error that says equations were not cleaned and is now stuck in limbo

**Comment by dvince2** [comment]
![Image](https://github.com/user-attachments/assets/6dec7773-5aae-4e7d-88b1-c3234057237c)
@liunelson I can't seem to reproduce this, and running model from equations on 'page 3' here works as expected I believe? This was perhaps addressed by another fix or maybe a test-day hiccup?

**Comment by liunelson** [comment]
I just tried to reproduce this bug but couldn't. You're probably right - test-day hiccup!

I do however get this error when using the MIRA service:
* ![Image](https://github.com/user-attachments/assets/4f848da5-b821-48d9-86e9-9e3012d7e02c)
* After the error shows up, the operator seems to be stuck in a limbo where clicking "Reset" or "Run" again doesn't do anything - the only way out is to create a new operator and delete this one.
* The operator should fail safely and let me try again with an edited set of equations, not just perpetually waiting for a response that never comes.

@dvince2 not sure if you want this adjacent bug to here or in a new issue.
